force-app/main/default/classes/RD2_StatusAutomationService.cls

```diff
@@ -89,6 +89,15 @@ public with sharing class RD2_StatusAutomationService {
         }
     }
 
+    public Boolean isSettingValid() {
+        return String.isBlank(validateSetting(
+            rd2Settings.statusAutomationNumberOfDaysForLapsed,
+            rd2Settings.statusAutomationNumberOfDaysForClosed,
+            rd2Settings.statusAutomationLapsedValue,
+            rd2Settings.statusAutomationClosedValue)
+        );
+    }
+
     public String validateSetting(Integer daysForLapsed, Integer daysForClosed, String lapsedStatus, String closedStatus) {
```

Comment:
Would it look cleaner if this just took a reference to `rd2Settings` instead of separate parameters?

Reply:
Now that I see it written out, I'm not sure if it actually looks cleaner 😆  I'll let the next reviewer decide!